css/style.css

```diff
@@ -882,3 +860,157 @@ body {
     font-size: 1.25rem;
   }
 }
+
+/******************* Popup Window Style ********************/
+
+.cntInfoPopup {
+  width: 90vw;
+  height: 85vh;
+  background: var(--color-background_clear);
+  margin: auto;
+  margin-top: 4rem;
+  padding: 1.5rem;
+}
+
+.btnClosePopup {
+  z-index: 99;
+  width: 2rem;
+  height: 2rem;
+  margin: 1rem;
+  text-align: center;
+  cursor: pointer;
+  float: right;
+}
+
+.popup-col1 {
+  text-align: center;
+}
+
+.popupImgCnt {
+  width: auto;
+}
+
+.PopupImg {
+  width: 100%;
+  max-width: 500px;
+}
```

Comment:
- [ ] The content of the card is overflowing from the popup in the desktop version. Please make sure the popup is responsive between the two required screen sizes i.e. (`Desktop: >=768px`) and `mobile: >= 375px`

### Overflowing content:

<img width="1201" alt="Screenshot 2023-04-19 at 9 49 08 PM" src="https://user-images.githubusercontent.com/44798044/233148897-a09d8470-5ff6-4605-b27e-a4c4d9700398.png">

Reply:
Thank you @shella12 , Adding the scroll function this also was solved. great thanks a lot👍